config/crd/patches/cainjection_in_spannerautoscaleschedules.yaml

```diff
@@ -0,0 +1,7 @@
+# The following patch adds a directive for certmanager to inject CA into the CRD
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
```

Comment:
It looks like you need to add `vars` to config/crd/kustomization.yaml like config/default/kustomization.yaml.

Reply:
Yes, `make install` (only installs the CRD) works, but `make uninstall install` does not work because of the missing variables currently. I think this is a scaffolding issue in the Kubebuilder.
As a work-around, I have been using `make undeploy deploy` (re-installs the CRD as well as all the other resources) or just re-create the Kind cluster sometimes.

To fix this, I will try to find if it is a scaffolding issue or just a documentation issue in Kubebuilder.

For our case, the webhook manifests for `SpannerAutoscaleSchedule` have not been generated yet. When I enable the webhooks for `SpannerAutoscaleSchedule`, then I will add these variables too in our configs.